src/user-icon/user-icon.scss

```diff
@@ -43,9 +45,18 @@
     text-decoration: none;
     text-transform: uppercase;
     background-color: $iui-color-dataviz-celery;
+    overflow: hidden;
     @include themed {
       color: t(iui-color-foreground-accessory);
     }
+
+    > .iui-icon {
```

Comment:
Why SVGs are placed differently than img? I don't think we can handle that in React properly.

Reply:
I've updated the code so the `<svg>` is placed in the same location as the `<img>`.  The only other difference is the `<abbr>` changes to a `<div>` for the SVG user icons.

Reply:
Why does `abbr` change to `div`?

Also, cant you use `display: inline-flex` to center the svg? Instead of this hacky transform

Reply:
I replaced the `abbr` with `div` because I thought it made sense, we are no longer using that element to display initials.  I suppose it wouldn't hurt to stay as `abbr` and display no content within it. I'll change them back to `abbr`.

Regarding the `inline-flex`, I tried that plus removing `absolute` from `.iui-icon` and am getting something like this.  Any idea why?
![Screen Shot 2021-06-22 at 6 14 30 PM](https://user-images.githubusercontent.com/849817/123006482-b5415180-d385-11eb-9e46-7268329e0e5f.png)

Reply:
I think the `position: absolute` needs to stay